tests/integration/test_distributed_inter_server_secret/test.py

```diff
@@ -26,15 +26,6 @@ def make_instance(name, cfg, *args, **kwargs):
 # _n1/_n2 contains cluster with different <secret> -- should fail
 n1 = make_instance("n1", "configs/remote_servers_n1.xml")
 n2 = make_instance("n2", "configs/remote_servers_n2.xml")
-backward = make_instance(
-    "backward",
-    "configs/remote_servers_backward.xml",
-    image="clickhouse/clickhouse-server",
-    # version without DBMS_MIN_REVISION_WITH_INTERSERVER_SECRET_V2
-    tag="22.6",
```

Comment:
@Algunenano I've resubmitted this test - https://github.com/ClickHouse/ClickHouse/pull/63070 (23.3 wasn't that long ago or otherwise it should be cleaned up further)